cuda_bindings/cuda/bindings/_internal/nvjitlink_windows.pyx

```diff
@@ -11,19 +11,71 @@ from .utils import FunctionNotFoundError, NotSupportedError
 
 from cuda.pathfinder import load_nvidia_dynamic_lib
 
-import win32api
+from libc.stddef cimport wchar_t
+from libc.stdint cimport uintptr_t
+from cpython cimport PyUnicode_AsWideCharString, PyMem_Free
+
+from .utils import NotSupportedError
+
+cdef extern from "windows.h" nogil:
```

Comment:
Can we use `cuda.bindings._lib.windll` instead of redefining these?

Reply:
This is from a different codegen. Mike has made it such that this extern block is autogenerated.

Reply:
The codegen in cybind has to work for modules that ship with cuda_bindings and those that don't, which is why I arrived at this.  I think we can iterate on making that a bit better over time...